rkt/pods.go

```diff
@@ -733,14 +734,33 @@ func (p *pod) getState() string {
 
 // getPID returns the pid of the pod.
 func (p *pod) getPID() (pid int, err error) {
+	// check the "pid" (process ID) file if it exists, and on "ppid"
+	// (parent process ID). Recent stage1 uses "ppid" but we still
+	// support older rkt versions or alternative stage1 implementations.
```

Comment:
Ok. I should add some documentation in the [Stage1 ACI implementor's guide](https://github.com/coreos/rkt/blob/master/Documentation/devel/stage1-implementors-guide.md#rkt-enter--coreoscomrktstage1enter). At the moment, it is out of sync since #826.